simul/lib/config.go

```diff
@@ -74,6 +74,8 @@ type RunConfig struct {
 	Threshold int
 	// Number of failing nodes
 	Failing int
+	// Number of processes/instances for this run
+	NbOfProc int
```

Comment:
Sorry for the nitpick but I can't help it. Can we remove the `NbOf` in front ? we already know it's a number of something...^^
Then you could put "Processes" since in both cases it corresponds to the number of processes we run (but I'm open to discussion on that!!)

Reply:
Done:
https://github.com/ConsenSys/handel/pull/67/commits/d5b3884f0bc150b00bee1ef1e81e3b4919cb4f33